modules/virt-configuring-huge-pages-for-vms.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/virt-using-huge-pages-with-vms.adoc
+
+[id="virt-configuring-huge-pages-for-vms_{context}"]
+= Configuring huge pages for virtual machines
+
+You can configure virtual machines to use pre-allocated huge pages by including the
+`memory.hugepages.pageSize` and memory request parameters in your virtual machine configuration.
+
+The memory request must be divisible by the page size. For example, you can not request `500Mi` memory with a page size of `1Gi`.
+
+[NOTE]
```

Comment:
@kbidarkar
@stu-gott provided some draft text that hopefully clarifies the issue in BZ#[1822875](https://bugzilla.redhat.com/show_bug.cgi?id=1822875). If this is acceptable then I can merge this and put a link to the doc in the two bugs